src/Fetch.js

```diff
@@ -487,7 +487,7 @@ function emscripten_start_fetch(fetch, successcb, errorcb, progresscb) {
 #if FETCH_DEBUG
     console.log('fetch: operation success. e: ' + e);
 #endif
-    if (onsuccess && typeof dynCall === 'function') Module['dynCall_vi'](onsuccess, fetch);
+    if (onsuccess) Module['dynCall_vi'](onsuccess, fetch);
```

Comment:
why not use makeDynCall here? I guess because this isn't linked in normally?

lgtm

Reply:
Yeah, Fetch.js is being shared with the worker creation, so could not use it here directly.